core/src/components/tab-button/tab-button.md.scss

```diff
@@ -35,7 +37,7 @@
 }
 
 :host(.tab-selected) .tab-button-text {
-  transform: $tab-button-md-text-transform-active;
+  --text-transform: #{$tab-button-md-text-transform-active};
```

Comment:
`--text-transform` is kind of confusing, since there is also `text-transform` which is completely different.